runner/main.js

```diff
@@ -266,6 +274,7 @@ class XMLHttpRequest {
             handleResponse({ error: "notfound" });
           } else {
             console.log(error);
+            console.trace();
```

Comment:
Is this line necessary or was it just put in temporarily and not removed?

Reply:
It isn't necessary so much as it was put in to be consistent with the other errors reporting traces.